RELEASE.md

```diff
@@ -20,6 +28,7 @@
 * Added example to explain how dataset factories work.
 
 ## Community contributions
+Many thanks to the following Kedroids for contributing PRs to this release:
```

Comment:
@ravi-kumar-pilla could you double check in the list of merged PRs if we should acknowledge anybody else?

Reply:
Thank you @astrojuanlu for pointing out. Added other contributions now.